code/obj/machinery/manufacturer_subtypes.dm

```diff
@@ -0,0 +1,684 @@
+
+// Fabricator Defines
+
+/obj/machinery/manufacturer/general
+	name = "general manufacturer"
+	supplemental_desc = "This one produces tools and other hardware, as well as general-purpose items like replacement lights."
+	free_resources = list(/obj/item/material_piece/steel = 5,
+		/obj/item/material_piece/copper = 5,
+		/obj/item/material_piece/glass = 5)
+	available = list(/datum/manufacture/screwdriver,
+		/datum/manufacture/wirecutters,
+		/datum/manufacture/wrench,
+		/datum/manufacture/crowbar,
+		/datum/manufacture/extinguisher,
+		/datum/manufacture/welder,
+		/datum/manufacture/flashlight,
+		/datum/manufacture/weldingmask,
+		/datum/manufacture/metal,
+		/datum/manufacture/metalR,
+		/datum/manufacture/rods2,
+		/datum/manufacture/glass,
+		/datum/manufacture/glassR,
+		/datum/manufacture/atmos_can,
+		/datum/manufacture/gastank,
+		/datum/manufacture/miniplasmatank,
+		/datum/manufacture/minioxygentank,
+		/datum/manufacture/player_module,
+		/datum/manufacture/cable,
+		/datum/manufacture/powercell,
+		/datum/manufacture/powercellE,
+		/datum/manufacture/powercellC,
+		/datum/manufacture/powercellH,
+		/datum/manufacture/light_bulb,
+		/datum/manufacture/red_bulb,
+		/datum/manufacture/yellow_bulb,
+		/datum/manufacture/green_bulb,
+		/datum/manufacture/cyan_bulb,
+		/datum/manufacture/blue_bulb,
+		/datum/manufacture/purple_bulb,
+		/datum/manufacture/blacklight_bulb,
+		/datum/manufacture/light_tube,
+		/datum/manufacture/red_tube,
+		/datum/manufacture/yellow_tube,
+		/datum/manufacture/green_tube,
+		/datum/manufacture/cyan_tube,
+		/datum/manufacture/blue_tube,
+		/datum/manufacture/purple_tube,
+		/datum/manufacture/blacklight_tube,
+		/datum/manufacture/table_folding,
+		/datum/manufacture/jumpsuit,
+		/datum/manufacture/shoes,
+#ifdef UNDERWATER_MAP
+		/datum/manufacture/flippers,
+#endif
+		/datum/manufacture/breathmask,
+#ifdef MAP_OVERRIDE_NADIR
+		/datum/manufacture/nanoloom,
+		/datum/manufacture/nanoloom_cart,
+#endif
+		/datum/manufacture/fluidcanister,
+		/datum/manufacture/meteorshieldgen,
+		/datum/manufacture/shieldgen,
+		/datum/manufacture/doorshieldgen,
+		/datum/manufacture/patch,
+		/datum/manufacture/saxophone,
+		/datum/manufacture/trumpet)
+	hidden = list(/datum/manufacture/RCDammo,
+		/datum/manufacture/RCDammomedium,
+		/datum/manufacture/RCDammolarge,
+		/datum/manufacture/bottle,
+		/datum/manufacture/vuvuzela,
+		/datum/manufacture/harmonica,
+		/datum/manufacture/bikehorn,
+		/datum/manufacture/bullet_22,
+		/datum/manufacture/bullet_smoke,
+		/datum/manufacture/stapler,
+		/datum/manufacture/bagpipe,
+		/datum/manufacture/fiddle,
+		/datum/manufacture/whistle)
+
+#define WIRE_MALF 3 //! wire which causes machine to malfunction
+#define MALFUNCTION_WIRE_CUT 15 & ~(1<<WIRE_MALF)
```

Comment:
i don't get this, why is this here and only used once

Reply:
Can def swap it over in favor of a comment, was a hotfix for bringing over the file